runtime/testy/studentCode.py

```diff
@@ -8,10 +8,35 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  print("setup")
```

Comment:
Is there a reason this is going to be merged in? If it's for examples, this should be in a separate file documenting how to write this code rather than merging this into master.